### PR TITLE
Add destination api

### PR DESCRIPTION
**Pull request by sreerajmp**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

Add destination api 
## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

File: src/destinations/destination.repository.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { EntityRepository, Repository } from 'typeorm';
+import {Destination} from './entities/destination.entity';
+import {AddDestinationDto } from './dto/addDestination.dto';
+import { v4 as uuidv4 } from 'uuid';
+import {
+  InternalServerErrorException,
+} from '@nestjs/common';
+@EntityRepository(Destination)
```
Comment (kuism) [inline src/destinations/destination.repository.ts:8]: leave a line here

File: src/destinations/destinations.service.ts [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+import {DestinationRepository} from 'src/destinations/destination.repository'
 
 @Injectable()
-export class DestinationsService {}
+export class DestinationsService {
+    constructor(  
+    @InjectRepository(DestinationRepository)
+    private DestinationRepository : DestinationRepository){}
+
+    async addDestination(AddDestinationDto:AddDestinationDto):Promise <any>{
```
Comment (kuism) [inline src/destinations/destinations.service.ts:12]: change it to createDestination

File: src/destinations/destinations.service.ts [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+        console.log("log:",AddDestinationDto)
```
Comment (kuism) [inline src/destinations/destinations.service.ts]: remove console.log

**Review by kuism — COMMENTED** [review]
minor changes

File: src/destinations/destinations.service.ts (hunk 3 above)
Comment (sreerajmp) [reply src/destinations/destinations.service.ts]: Done please review

**Review by kuism — APPROVED** [review]

**Pull request merged**